### PR TITLE
Feat/team menu

### PR DESCRIPTION
**Pull request by MamounKolovos**

### CODE REVIEW

File: client/src/objects/player.js
Comment (leia-uwu) [inline client/src/objects/player.js:1]: why this change

File: client/src/game.js [hunk 1]
```diff
@@ -967,6 +967,7 @@ export class Game {
         }
         // Update player status
         if (msg.playerStatusDirty) {
+            // console.log("hello");
```
Comment (leia-uwu) [inline client/src/game.js]: :p remove this pls

File: shared/gameConfig.ts
Comment (leia-uwu) [inline shared/gameConfig.ts:1]: should be on server config since its only used by the server and not by the client

File: client/src/objects/player.js
Comment (MamounKolovos) [reply client/src/objects/player.js:1]: player indicators on the minimap fade if i keep it uncommented

File: shared/gameConfig.ts
Comment (lmssiehdev) [reply shared/gameConfig.ts:1]: I will be using it here as `[TeamMode.Solo]: "solo"`

https://github.com/leia-uwu/resurviv/blob/0a85a99759b0d4c7ea5bbab651660ecb3b4bd557/client/src/siteInfo.js#L30-L35

File: client/src/objects/player.js
Comment (leia-uwu) [reply client/src/objects/player.js:1]: i think theres something wrong with the server code then because that code is from the original client, they shouldn't fade
